locodedb/calls.go

```diff
@@ -0,0 +1,94 @@
+package locodedb
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+var errRecordNotFound = errors.New("record not found")
+
+type LOCODE [2]string
+
+var (
+	mCountries map[CountryCode]string
```

Comment:
used for reading but not used for writing, always empty

Reply:
`locodedb` package is basically read-only.

Reply:
@carpawell  it filles in InitLocodeData() i think

Reply:
@AliceInHunterland, yes, you have fixed it, now it is ok